packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,63 @@
+$default-border: 1px solid #1c1f21;
+
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #fff);
+    border-bottom: var(--terra-action-header-border-bottom, $default-border);
+    border-left: var(--terra-action-header-border-left, $default-border);
```

Comment:
Setting the left, right, and top border to `1px solid #1c1f21` seems odd to me as it doesn't match the design of the [action header in clinical](http://engineering.cerner.com/terra-ui/#/site/components/clinical/site/action-header/index) or the [header in the dialog](http://engineering.cerner.com/terra-ui/#/site/components/core/site/dialog/index). 

I think these styles should be updated to align more with the dialog header / clinical action header:

```css
background-color: var(--terra-action-header-background-color, #f4f4f4);
border-bottom: var(--terra-action-header-border-bottom, #dedfe0);
border-left: var(--terra-action-header-border-left);
border-right: var(--terra-action-header-border-right);
border-top: var(--terra-action-header-border-top);
```

cc/ @neilpfeiffer

Reply:
I set these based off the "Standard Implementation" callouts in Zeplin.

Reply:
Ahh, I understand the confusion -- @ChaseSinclair the "Standard Implementation" board is only for details of how the component is constructed and how it behaves. That board is not for Style or CSS. If you look at the `Styles: Themeable` & `Styles: Controlled` on that same board - it correctly lists only border-bottom.

What you should have used was details on the Terra-opensource-theme board (which currently is blank and wasn't provided in this case). But those details / colors are what already exist in the terra-clinical component, which @bjankord has included above.

Reply:
Updated styles to match terra-clinical-action-header [here](https://github.com/cerner/terra-core/commit/85b35f2f932e64ced0e37458cd33fde2f0633fe7).